src/yunohost/domain.py

```diff
@@ -61,7 +61,11 @@ def domain_list(exclude_subdomains=False):
                 continue
         result_list.append(domain)
 
-    return {'domains': result_list}
+    result_list.sort()
+    return {
+        'domains': result_list,
+        'default': result_list.index(_get_maindomain())
```

Comment:
I think this index research should be done in web admin to improve cli display AND avoid postinstall failure (it search the index of 'yunohost.org' which is not in domains list).
```suggestion
        'default': _get_maindomain()
```